Makefile

```diff
@@ -12,57 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Please note that this file was generated from [terraform-google-module-template](https://github.com/terraform-google-modules/terraform-google-module-template).
+# Please make sure to contribute relevant changes upstream!
+
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-# All is the first target in the file so it will get picked up when you just run 'make' on its own
-all: check_shell check_python check_golang check_terraform check_docker check_base_files test_check_headers check_headers check_trailing_whitespace generate_docs
-
-# The .PHONY directive tells make that this isn't a real target and so
-# the presence of a file named 'check_shell' won't cause this target to stop
-# working
-.PHONY: check_shell
-check_shell:
-	@source test/make.sh && check_shell
-
-.PHONY: check_python
-check_python:
-	@source test/make.sh && check_python
-
-.PHONY: check_golang
-check_golang:
-	@source test/make.sh && golang
-
-.PHONY: check_terraform
-check_terraform:
-	@source test/make.sh && check_terraform
-
-.PHONY: check_docker
-check_docker:
-	@source test/make.sh && docker
-
-.PHONY: check_base_files
-check_base_files:
-	@source test/make.sh && basefiles
-
-.PHONY: check_shebangs
-check_shebangs:
-	@source test/make.sh && check_bash
-
-.PHONY: check_trailing_whitespace
-check_trailing_whitespace:
-	@source test/make.sh && check_trailing_whitespace
-
-.PHONY: test_check_headers
-test_check_headers:
-	@echo "Testing the validity of the header check"
-	@python test/test_verify_boilerplate.py
-
-.PHONY: check_headers
-check_headers:
-	@echo "Checking file headers"
-	@python test/verify_boilerplate.py
-
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.1.0
+DOCKER_IMAGE_DEVELOPER_TOOLS := cft/developer-tools
+REGISTRY_URL := gcr.io/cloud-foundation-cicd
+
+# Enter docker container for local development
+.PHONY: docker_run
+docker_run:
+	docker run --rm -it \
+		-e SERVICE_ACCOUNT_JSON \
+		-v $(CURDIR):/workspace \
+		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
+		/bin/bash
+
+# Execute lint tests within the docker container
```

Comment:
This file is missing commands for integration testing.

Please just make a direct copy from the template and explain any divergences.

Reply:
@morgante, I thought you requested intermediate version for Cloud Build integration with current bats tests. Supporting both ways of integration testing: dynamically generated resources and static/placeholders in the bats version of it would increase already significant level of duplicates in the code base. I suggest logging another issue for tests standardization to be processed according to your priorities.

Reply:
To be clear, continuing with bats support is fine.

You need to include a command for running bats tests in the Makefile though. Take a look at how this was done on another module: https://github.com/terraform-google-modules/terraform-google-startup-scripts/pull/29/files

Reply:
> To be clear, continuing with bats support is fine.
> 
> You need to include a command for running bats tests in the Makefile though. Take a look at how this was done on another module: https://github.com/terraform-google-modules/terraform-google-startup-scripts/pull/29/files

@morgante [the script](https://github.com/terraform-google-modules/terraform-google-startup-scripts/pull/29/files#diff-b67911656ef5d18c4ae36cb6741b7965R65) [calls kitchen](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/concourse/build/developer-tools/build/scripts/task_helper_functions.sh#L408-L410) to run [kitchen-based](https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/kitchen.yml#L31-L43) tests, not bats. The migration to kitchen supposed to be a next step as far as I understood the original plan. I'll make a make a temporary target for bats test.